informer/collector.go

```diff
@@ -41,21 +53,39 @@ func (i *Informer) Collect(ch chan<- prometheus.Metric) {
 	for {
 		select {
 		case event, ok := <-watcher.ResultChan():
```

Comment:
This is out of scope of this PR, but I'd like to discuss that.

Thanks to @JosephSalisbury I learnt recently more about the prometheus collectors.
This makes me think if `Watch` api is correct one to use here. I'd rather watch and update the metrics as objects arrive OR list all the objects here with `List` call.

Isn't what we do here a bit odd?

Reply:
I'm not certain how much difference it makes. Would need investigation.